src/onegov/org/models/document_form.py

```diff
@@ -0,0 +1,146 @@
+from __future__ import annotations
+from uuid import uuid4
+
+from onegov.chat.models.message import associated
+from onegov.core.collection import GenericCollection
+from onegov.core.orm import Base
+from onegov.core.orm.mixins import (
+    ContentMixin, TimestampMixin, content_property, dict_property)
+from onegov.core.orm.mixins.content import dict_markup_property
+from onegov.core.orm.types import UUID
+from onegov.core.utils import normalize_for_url
+from onegov.file import AssociatedFiles, File, SearchableFile
+from onegov.form import FormCollection
+from onegov.org.observer import observes
+from onegov.reservation import ResourceCollection
+from onegov.org.models import AccessExtension
+from onegov.search import SearchableContent
+from sqlalchemy import Column, Text
+
+
+from typing import Any, TYPE_CHECKING
+if TYPE_CHECKING:
+    import uuid
+    from sqlalchemy.orm import Query, Session
+
+
+class DocumentFormFile(File, SearchableFile):
+
+    __mapper_args__ = {'polymorphic_identity': 'document_form_file'}
+
+    es_type_name = 'document_form_file'
+
+    @property
+    def es_public(self) -> bool:
+        return True
+
+
+class FormDocument(Base, ContentMixin, TimestampMixin, AccessExtension,
+                   SearchableContent, AssociatedFiles):
+
+    __tablename__ = 'form_documents'
+
+    es_properties = {
+        'title': {'type': 'localized'},
+        'lead': {'type': 'localized'},
+    }
+
+    #: An internal id for references (not public)
+    id: Column[uuid.UUID] = Column(
+        UUID,  # type:ignore[arg-type]
+        primary_key=True,
+        default=uuid4
+    )
+
+    #: A nice id for the url, readable by humans
+    name: Column[str] = Column(Text, nullable=False, unique=True)
+
+    title: Column[str] = Column(Text, nullable=False)
+
+    #: Describes the document briefly
+    lead: dict_property[str | None] = content_property()
```

Comment:
This one should probably be a meta property
```suggestion
    lead: dict_property[str | None] = meta_property()
```

For pages we care less, since we cache them in a different format for things like navigation. But for other things we want everything we'd use in a list view to be a `meta_property`, rather than a `content_property`.